spec/lumberjack/client_spec.rb

```diff
@@ -35,31 +36,6 @@
         expect(socket.sequence).to eq(1)
       end
     end
-
-    context "ack" do
-
-      let(:hash)   { {:a => 1, :b => 2}}
-
-      before(:each) do
-        allow(socket).to receive(:write).and_return(true)
-      end
-
-      it "increments the sequence per windows size" do
-        allow(socket).to receive(:read_version_and_type).and_return([1, 'A'])
-        expect(socket).to receive(:ack).twice.and_call_original
-
-        [5000, 10000].each do |last_ack|
-          windows_size = 5001
-
-          allow(socket).to receive(:read_last_ack).and_return(last_ack)
-
-          windows_size.times do
-            socket.write_hash(hash)
-          end
-        end
-
-      end
-    end
   end
 
   describe Lumberjack::Encoder do
```

Comment:
The test is better in the integration test, I've removed this one since It involved a lot of stubbing.